### DISCUSSION TITLE
MPV怎么流畅播放直播源吗？

### DISCUSSION THREAD

**Discussion by xovow** [hooke007/mpv_PlayKit#249] [mpv 通用]
比如这个项目里的直播源https://github.com/imDazui/Tvlist-awesome-m3u-m3u8
我用MPV打开m3u文件播放几秒钟就要缓冲

**Comment by yeezylife** [comment]
当然能播呀 要缓冲是源不行

**Comment by hooke007** [comment]
都能缓冲了那还叫不能吗

**Reply by xovow** [reply]
那怎么让它流畅地播放呢，这样几秒缓冲一次相当于看不了啊

**Reply by hooke007** [reply]
首先你要确定是mpv的问题还是网络的问题还是源的问题

**Reply by hooke007** [reply]
试过vlc了，一样不行，至少说明不是程序的问题
